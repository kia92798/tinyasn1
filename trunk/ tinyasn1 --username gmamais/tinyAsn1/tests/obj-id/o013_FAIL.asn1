MY-MODULE DEFINITIONS AUTOMATIC TAGS::= BEGIN
	OI ::= OBJECT IDENTIFIER	
	OI2 ::= OI	

	oi2 OI2 ::= {MY-MODULE2.oi1 MY-MODULE2.myIntVar}
END

MY-MODULE2 DEFINITIONS AUTOMATIC TAGS::= BEGIN
	myIntVar INTEGER ::= 5
	oi1 OBJECT IDENTIFIER ::= {itu-t recommendation bb}
END