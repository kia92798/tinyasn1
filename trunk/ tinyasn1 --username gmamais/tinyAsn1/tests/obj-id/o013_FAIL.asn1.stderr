Error in line: 10, col:49. Unknown identifier: bb
